### DISCUSSION TITLE
[2022/07/26] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#215] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/214 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by izumin5210** [comment]
↓をふまえつつ tailwind について話をした

- https://blog.lacolaco.net/2022/07/two-sides-of-tailwindcss/
- https://twitter.com/takanoripe/status/1550393373558992896

**Comment by izumin5210** [comment]
chromatic による UI Test の話と、MSW + Play function と Chromatic の UI Test が相性いい話をした
